File: level_1/PYTHON/소수 찾기.py
def solution(n):
    a=set([i for i in range(3,n+1,2)])
    for i in range(3,n+1,2):
        if i in a:
            a-=set([i for i in range(i+i,n+1,i)])
    return len(a)+1

File: level_1/C++/소수 찾기.cpp
#include <vector>
using namespace std;

int solution(int n) {
	int answer = 0;
	vector<int> tmp;
	for (int i = 0; i<n; i++) {
		tmp.push_back(i);
	}

	for (int i = 0; i<n + 1; i++) {
		if (tmp[i] == 0 || tmp[i] == 1) continue;

		for (int z = i; z<n + 1; z += i) {
			tmp[z] = 0;
		}
		answer++;
	}
	return answer;
}